src/TLuaInterpreter.cpp

```diff
@@ -3206,8 +3238,7 @@ int TLuaInterpreter::setFont(lua_State* L)
     QString font{lua_tostring(L, s)};
 
     if (!mudlet::self()->getAvailableFonts().contains(font, Qt::CaseInsensitive)) {
-        lua_pushnil(L);
-        lua_pushfstring(L, "font '%s' is not available)", font.toUtf8().constData());
```

Comment:
nice hard to spot bug again eliminated in passing by standardizing (the message ended in ) but should not)

Reply:
To avoid confusion would it be better to change `font` to `fontName`?

Reply:
That is discussed in #4638

Reply:
I don't think anyone would actually really be so confused about this that they'd make an error to be honest